source/helpers.coffee

```diff
@@ -6,12 +6,19 @@ defineTypeWithFields = (BaseType, namespace, className, fields) ->
   SubType = BaseType.extend(classPath)
   SubType::fields = -> _.extend BaseType::fields(), fields
   Space.resolvePath(namespace)[className] = SubType
+  definedSerializable = {}
+  definedSerializable[classPath] = SubType
+  return definedSerializable
```

Comment:
I'd just return the SubType (make it explicit), and let `Space.messaging.define` handle the object